docs/concepts/queries.md

```diff
@@ -68,32 +68,11 @@ Because materialized views are incrementally updated as new events arrive,
 pull queries run with predictably low latency. They're a great match for
 request/response flows. For asynchronous application flows, see
 [Push Query](/developer-guide/ksqldb-reference/select-push-query/).
+Pull queries are expressed using a strict subset of ANSI SQL.
 
 Execute a pull query by sending an HTTP request to the ksqlDB REST API, and
 the API responds with a single response.
 
-### Pull query features and limitations
+If you want to learn more about Pull Queries, see [Pull Queries](select-pull-query.md).
 
-- Pull queries are expressed using a strict subset of ANSI SQL.
-- You can execute pull queries on any derived table created by using the
-  CREATE TABLE AS SELECT statement.
-- For non-windowed aggregations, pull queries only support looking up events
-  by key.
-- WHERE clauses must have constraints that encompass all key column(s) for non-windowed tables.
 
-- In addition, windowed tables support bounds on `WINDOWSTART` and `WINDOWEND` using operators
-  `<=`, `<`, `=`, `>`, `>=`.
-- JOIN, PARTITION BY, GROUP BY and WINDOW clauses aren't supported.
```

Comment:
Do we mention this in the other doc?

Reply:
We implicitly mention this by saying
```
Synopsis:
SELECT select_expr [, ...]
  FROM table
  [ WHERE where_condition ]
  [ AND window_bounds ];
  ```

Reply:
I am explicitly adding some of this info in the other doc as well

Reply:
done